include/aspect/material_model/rheology/friction_models.h

```diff
@@ -76,7 +77,8 @@ namespace aspect
           double
           compute_friction_angle(const double current_edot_ii,
                                  const unsigned int volume_fraction_index,
-                                 const double static_friction_angle) const;
+                                 const double static_friction_angle,
+                                 const Point<dim> &position) const;
```

Comment:
If we want to add more options in the future that need other input parameters we need to find a different solution than to keep adding parameters here, but for now it's fine. Do you already have plans to add other options?

Reply:
Yes, I see. To include rate and state eventually, I will need the following inputs as well: 

```
                                       const std::vector<double> &composition, 
                                       typename DoFHandler<dim>::active_cell_iterator current_cell,
```
Should I then add these already?